README.md

```diff
@@ -9,13 +9,151 @@ DUH component export to Scala
 npm i duh-scala
 ```
 
+To export a scala blackbox wrapper and attach method
 ```bash
 duh-export-scala <mycomp>.json5 -o <output-dir>
 ```
 
+To export a `RegisterRouter` corresponding the the `memoryMap`s in a duh component
+```bash
+duh-export-regmap <mycomp>.json5 -o <output-dir>
+```
+
+To export a scala blackbox wrapper and attach method for a monitor
+```bash
+duh-export-monitor <mycomp>.json5 -o <output-dir>
```

Comment:
Maybe a short explanation what a blackbox wrapper, RegisterRouter, and a monitor are.